react/react1/week2/todo-app/src/App.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+import Timer from './components/Timer';
+import Title from './components/Title';
+import TodoList from './components/TodoList';
+
+const Wrapper = styled.div`
+  align-items: center;
+  background: #d5ffe3;
+  display: flex;
+  flex-direction: column;
+  font-size: 2rem;
+`;
+
```

Comment:
It is very great to see that you have started working with `styled-components`, that is a very great choice. well done!